src/libraries/System.Private.CoreLib/tests/math/MathFTests.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace System
+{
+    class MathTests
+    {
+        public struct ScaleBTest
```

Comment:
Why are we not using xunit tests in System.Runtime.Tests?

Reply:
Saw your comment above - take a look at https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime.Extensions/tests/System/MathF.cs

Reply:
This is exactly what I was looking for before! I just missed it when I was looking for current tests. I'll just add my new test cases to the existing `ScaleB` tests.